<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgDaySelector</class>
 <widget class="QWidget" name="WdgDaySelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbMonday">
     <property name="text">
      <string>Monday</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="cbSaturday">
     <property name="text">
      <string>Saturday</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cbTuesday">
     <property name="text">
      <string>Tuesday</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cbSunday">
     <property name="text">
      <string>Sunday</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cbWednesday">
     <property name="text">
      <string>Wednesday</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cbHoliday">
     <property name="text">
      <string>Holiday</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="cbThursday">
     <property name="text">
      <string>Thursday</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cbSchool">
     <property name="text">
      <string>School</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="cbFriday">
     <property name="text">
      <string>Friday</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="cbVacation">
     <property name="text">
      <string>Vacation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
